src/services/sync/strategies/CSVSyncer.js

```diff
@@ -50,8 +51,13 @@ class CSVSyncer extends mix(Syncer).with(ClassImporter) {
 		super(app, stats, logger);
 
 		this.options = options;
-		this.requestParams = requestParams;
-
+		this.requestParams = {
+			...requestParams,
+			headers: {
+				'x-api-key': Configuration.get(CLIENT_API_KEY),
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['CLIENT_API_KEY' is not defined.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5416140)